ch10.md

```diff
@@ -107,20 +106,16 @@ Again, because we're using partial application to achieve this result, we must e
 Let's look at another example.
 
 ```js
-// Helpers:
-// ==============
-//  $ :: String -> IO DOM
-var $ = function(selector) {
-  return new IO(function() { return document.querySelector(selector)});
-}
+//-- Helpers ----------------------------------------------------------
+//    $ :: String -> IO DOM
+const $ = selector => new IO(_ => document.querySelector(selector))
 
-//  getVal :: String -> IO String
-var getVal = compose(map(_.prop('value')), $);
```

Comment:
** reminder -> we need to find a neat way to refer to all these support functions.